Resources/Prototypes/_Goobstation/Changeling/Actions/changeling.yml

```diff
@@ -512,7 +512,7 @@
 - type: entity
   id: ActionLastResort
   name: Last Resort
-  description: Abandon your current body and escape in the form of a headslug. Costs 20 chemicals.
+  description: We abandon our current body, placing us in control of a vessel that can plant our likeness in a new host. Costs 20 chemicals.
```

Comment:
this is a little inaccurate because headslug turns you into lesser form but idk if it matters like at all

Reply:
technically the headslug is the true form of the changeling iirc

Reply:
ohhh shit that's fun. so this could be smth like "We abandon our vessel, escaping in our extremely vulnerable but true self, to find a new body to occupy.."

Reply:
changed to `We abandon our vessel, escaping in our vulnerable true self, to find a new body to occupy.` - maybe i should make it a little more verbose?